### PR TITLE
Added github PR template

### PR DESCRIPTION
**Pull request by patphongs**
## Summary

- Resolves https://github.com/fecgov/openfec/issues/4787

_Adds github PR template per [reviewed drafted document](https://docs.google.com/document/d/1fBMFNUr6GFJCHwhHNOP-VTIcZ3CJR44sWUQxjkahoVY/edit#) 🔒._

### Required reviewers

One designer and one developer reviews are required prior to merge.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Only github PR templates

## Screenshots

Screen Shot 2021-03-22 at 9 40 35 AM

## Related PRs
Will list after those PRs are created

branch | PR
------ | ------
feature/4787-pr-template-changes (fec-cms) | [link](https://github.com/fecgov/fec-cms/pull/4489)
feature/4787-pr-template-changes (openfec) | [link](https://github.com/fecgov/openFEC/pull/4808)

## How to test
- You can view the screenshot above 
 
--OR--

- Copy the text from the PULL_REQUEST_TEMPLATE.md file
- Create a new issue in any repository and paste what was copied in the PULL_REQUEST_TEMPLATE.md file to the body of the new issue
- Click on the "Preview" tab to see what the pull request template will look like

____

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## How to test
+Include any information that may be helpful to the reviewer(s).
+_This might include:_
+
+- links to sample pages to test
+- Any local environmental setup that is unusual (env vars, API version to point to, etc)
+
+## System architecture updates (if applicable)
+_If this pull request changes our [current system diagram](https://github.com/fecgov/FEC/wiki/2.-FEC-system-diagram), include a description of those changes here and update the system diagram._
```
Comment (rfultz) [inline .github/PULL_REQUEST_TEMPLATE.md:36]: Should there be some kind of approval queue for updating the diagram? If one were to make a change and update the system diagram, but then change isn't accepted, the diagram has already been changed.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+------ | ------
+fix/other_pr | [link]()
+feature/other_pr | [link]()
+
+## How to test
+Include any information that may be helpful to the reviewer(s).
+_This might include:_
+
+- links to sample pages to test
+- Any local environmental setup that is unusual (env vars, API version to point to, etc)
```
Comment (rfultz) [inline .github/PULL_REQUEST_TEMPLATE.md:33]: Should we remind something like "_public_ env vars"?

**Review by rfultz — COMMENTED** [review]
Markdown linter wants the first headline to be a `#` but that seems really big for us. Markdown linter is also requesting one blank line before and after each headline.

🤷🏼‍♂️ ?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (patphongs) [reply .github/PULL_REQUEST_TEMPLATE.md]: I added an extra step to create a new ticket to update the diagram. But it's not contingent on merging the work.

**Comment by rfultz** [comment]
Looks good to me! @JonellaCulmer ?

**Review by JonellaCulmer — APPROVED** [review]
Looks good, thanks @patphongs

**Comment by patphongs** [comment]
Thanks @rfultz @JonellaCulmer, I'm going to go ahead and merge

**Pull request merged**